naresh/src/main/java/com/lftechnology/phpjava/assignmentthree/LargestPalindrome.java

```diff
@@ -0,0 +1,78 @@
+package com.lftechnology.phpjava.assignmentthree;
+
+
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.Scanner;
+
+/**
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ */
+public class LargestPalindrome implements MenuReader {
+
+    /**
+     * find the largest palindrome number
+     *
+     * @param startIndex
+     * @param endIndex
+     * @return
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    public long findLargestPalindrome(int startIndex, int endIndex) {
+        long palindrome = 0;
+        int max = Math.max(startIndex, endIndex);
+        int min = Math.min(startIndex, endIndex);
+        for (int i = max; i >= min; i--) {
+            for (int j = max; j >= min; j--) {
+                long product = i * j;
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Cast one of the operands of this multiplication operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS2184)